activerecord/test/cases/transaction_isolation_test.rb

```diff
@@ -2,14 +2,16 @@
 
 require "cases/helper"
 
-unless ActiveRecord::Base.connection.supports_transaction_isolation?
+unless ActiveRecord::Base.connection.supports_transaction_isolation? && !current_adapter?(:SQLite3Adapter)
   class TransactionIsolationUnsupportedTest < ActiveRecord::TestCase
     self.use_transactional_tests = false
 
     class Tag < ActiveRecord::Base
     end
 
     test "setting the isolation level raises an error" do
+      skip if current_adapter?(:SQLite3Adapter)
```

Comment:
Is this `skip` necessary?

Reply:
good catch, it's not. I forgot to remove it